src/managers/interfaces/IBalanceSheetManager.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity >=0.5.0;
+
+import {IERC165} from "src/misc/interfaces/IERC165.sol";
+
+interface IDepositManager is IERC165 {
```

Comment:
Idea of these interfaces is to make it easier for the UI, to support any balance sheet manager that uses these interfaces, rather than have to specifically support `OnOfframpManager`